entrypoint.sh

```diff
@@ -5,5 +5,6 @@ changelog_file=$2
 draft=$3
 write_to_summary=$4
 dry_run=$5
+repository=$6
 
-python create_release.py $github_token $changelog_file $draft $write_to_summary $dry_run
+python create_release.py $github_token $changelog_file $draft $write_to_summary $dry_run, $repository
```

Comment:
blocking: remove comma
```suggestion
python create_release.py $github_token $changelog_file $draft $write_to_summary $dry_run $repository
```